lib/ukfallocbuddy/tests/test_ukfallocbuddy.c

```diff
@@ -0,0 +1,179 @@
+#include <uk/test.h>
+#include <uk/fallocbuddy.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+
+struct uk_falloc *generate(void *fa_meta, void *start_zone, unsigned long len, __paddr_t *new_start);
+
+
+UK_TESTCASE(ukfallocbuddy, test_init)
+{
+	// test if the init function works
+	struct uk_falloc *fa;
+
+	fa = malloc(100000);
```

Comment:
I think we should use the `uk_fallocbuddy_size()` function when allocating the required memory for the
`struct uk_falloc` pointer, as specified [here](https://github.com/unikraft/unikraft/blob/staging/lib/ukfallocbuddy/include/uk/fallocbuddy.h#L48).

Reply:
Done.